src/mediastreambrokerobserver/MediaStreamBrokerObserver.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * [[MediaStreamBrokerObserver]] can be registered with a [[MediaStreamBroker]] to receive callbacks on media stream
+ * change events.
+ */
+export default interface MediaStreamBrokerObserver {
+  /**
+   * Called when video input stream are changed.
+   * @param videoStream: The new video input media stream
+   */
+  selectedVideoInputDidChanged?(videoStream: MediaStream | undefined): void;
```

Comment:
- nit: `Changed` => `Change` since you already have used `Did`.
  I don't like this Cocoa delegate pattern-inspired naming. It's useful when we notify an observer of a lifecycle, e.g., WillChange, DidChange, shouldChange, but we rarely provide many lifecycle observer methods.

- optional: We can also simplify naming by omitting `selected`. e.g., `videoInputDidChange`.